src/components/scrollBar/index.tsx

```diff
@@ -17,7 +17,7 @@ import View from '../view';
 import Image from '../image';
 
 
-interface Props extends FlatListProps<any> {
+interface Props extends FlatListProps<any>, ForwardRefInjectedProps {
```

Comment:
The props you're using for export should not include internal typings like `ForwardRefInjectedProps`.

`ForwardRefInjectedProps` typings are internal, we use them because we wrap our component with the `forwardRef` HOC. 

The way we do that is like this: 

```
interface Props {}

type InternalProps = Props & ForwardRefInjectedProps; 


// Here you use "InternalProps" cause you also expecting to get internal props like "forwardedRed", etc.. 
class YourComponent extends Component<InternalProps, State> {}

// Here you use "Props" - this is the API we expose for our users. 
export default asBaseComponent<Props, typeof YourComponent>(forwardRef<Props>(YourComponent));
```

Reply:
So what do I export? If I do this `export type ScrollBarProps = Props & ForwardRefInjectedProps;` I also export the internal prop, no?

Reply:
Yes, 
See my code snippet. 
I break the typings into two `Props` & `InternalProps`. Eventually I export `Props` only

Reply:
Done